app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/AcceptedListAdapter.java

```diff
@@ -36,11 +38,13 @@ public AcceptedListAdapter(List<Book> books, OnItemClickListener itemClickListen
 
     public static class ViewHolder extends BaseBookListAdapter.ViewHolder {
         final ImageButton moreBtn;
+        final TextView owner;
 
         public ViewHolder(View v, OnItemClickListener itemClickListener) {
             super(v);
+            owner = v.findViewById(R.id.item_book_owner);
             moreBtn = v.findViewById(R.id.item_book_more_btn);
-            moreBtn.setOnClickListener(this::showPopup);
+            moreBtn.setVisibility(View.GONE);
```

Comment:
it's better to just delete this and remove the showPopup() function instead of hiding the view